mne/io/eeglab/eeglab.py

```diff
@@ -253,15 +254,40 @@ def _set_dig_montage_in_init(self, montage):
         self.set_montage(montage + make_dig_montage(ch_pos=ch_pos, coord_frame="head"))
 
 
-def _handle_montage_units(montage_units):
-    n_char_unit = len(montage_units)
-    if montage_units[-1:] != "m" or n_char_unit > 2:
-        raise ValueError(
-            '``montage_units`` has to be in prefix + "m" format'
-            f', got "{montage_units}"'
-        )
+def _handle_montage_units(montage_units, eeg):
```

Comment:
This will give a better error message (and also uses one fewer line :) )
```suggestion
def _handle_montage_units(montage_units, eeg):
    _check_option("montage_units", montage_units, ("m", "dm", "cm", "mm", "auto"))
```